src/App.css

```diff
@@ -1,28 +1,19 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
-}
-
 .App-header {
   background-color: #222;
-  height: 150px;
   padding: 20px;
+  margin-top: 50px;
   color: white;
 }
 
 .App-title {
```

Comment:
Follow camelCase for naming the class names.
Also first letter should be small.